### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by JessieJaradal**

### CODE REVIEW

File: content/account-data-matching.md [hunk 1]
```diff
@@ -8,28 +8,28 @@ objectives:
 
 # TL;DR
 
-- Gumamit ng **mga pagsusuri sa pagpapatunay ng data** upang i-verify na tumutugma ang data ng account sa inaasahang halaga**.** Kung walang mga naaangkop na pagsusuri sa pagpapatunay ng data, maaaring gamitin ang mga hindi inaasahang account sa isang pagtuturo.
-- Upang ipatupad ang mga pagsusuri sa pagpapatunay ng data sa Rust, ihambing lang ang data na nakaimbak sa isang account sa inaasahang halaga.
+- Gumamitin ang **data validation checks** upang ma-verify na tumutugma ang data ng account sa inaasahang halaga**.** Kung walang mga naaangkop na data validations checks, maaaring gamitin ang mga hindi inaasahang account sa isang instruction.
```
Comment (kimerran) [inline content/account-data-matching.md]: @JessieJaradal I think `Gumamitin` is not a proper Filipino word

File: content/account-data-matching.md [hunk 2]
```diff
@@ -8,28 +8,28 @@ objectives:
...
+Maaari itong maging kapaki-pakinabang kapag ang mga account na kinakailangan ng isang instruction ay may mga dependencies sa mga halagang nakaimbak sa iba pang mga account o kung ang isang instruction ay naka-depende sa data na nakaimbak sa isang account.
 
-### Nawawalang pagsusuri sa pagpapatunay ng data
+### Nawawalang data validation check
 
 Ang halimbawa sa ibaba ay may kasamang tagubiling `update_admin` na nag-a-update sa field ng `admin` na nakaimbak sa isang `admin_config` na account.
 
-Ang tagubilin ay walang data validation check upang i-verify ang `admin` na account na pumipirma sa transaksyon ay tumutugma sa `admin` na nakaimbak sa `admin_config` account. Nangangahulugan ito ng anumang account na pumipirma sa transaksyon at ipinasa sa pagtuturo dahil maaaring i-update ng `admin` account ang `admin_config` account.
+Ang instruction ay ang nawawalang data validation check upang ma-verify ang `admin` na account na nag signing sa transaksyon ay tumutugma sa `admin` na nakaimbak sa `admin_config` account. Nangangahulugan ito ng anumang account na pumipirma sa transaksyon at ipinasa sa instruction dahil maaaring i-update ng `admin` account ang `admin_config` account.
```
Comment (kimerran) [inline content/account-data-matching.md]: `nag signing` should be `nag-sign`, right? :)

File: content/account-data-matching.md [hunk 3]
```diff
@@ -106,11 +106,11 @@ pub struct AdminConfig {
...
+Pinapasimple ito ng Anchor gamit ang constraints na `has_one`. Maaari mong gamitin ang constraints na `has_one` upang ilipat ang data validation check mula sa instruction logic patungo sa struct ng `UpdateAdmin`.
 
-Sa halimbawa sa ibaba, ang `has_one = admin` ay tumutukoy na ang `admin` account na pumipirma sa transaksyon ay dapat na tumugma sa `admin` na field na nakaimbak sa `admin_config` account. Upang magamit ang hadlang na `may_isa`, ang convention ng pagbibigay ng pangalan ng field ng data sa account ay dapat na pare-pareho sa pagbibigay ng pangalan sa struct ng pagpapatunay ng account.
+Sa halimbawa sa ibaba, ang `has_one = admin` ay tumutukoy na ang `admin` account na nag-signing sa transaksyon ay dapat na tumugma sa `admin` na field na nakaimbak sa `admin_config` account. Upang magamit ang constraints na `has_one`, ang convention ng pagbibigay ng pangalan ng field ng data sa account ay dapat na konsistent  sa pagbibigay ng pangalan sa struct ng pagpapatunay ng account.
```
Comment (kimerran) [inline content/account-data-matching.md]: `nag-signing` should be `nag-sign`

File: content/account-data-matching.md [hunk 4]
```diff
@@ -145,7 +145,7 @@ pub struct AdminConfig {
 }
 ```
 
-Bilang kahalili, maaari mong gamitin ang `constraint` upang manu-manong magdagdag ng isang expression na dapat suriin sa true upang magpatuloy ang pagpapatupad. Ito ay kapaki-pakinabang kapag sa ilang kadahilanan ay hindi maaaring maging pare-pareho ang pagbibigay ng pangalan o kapag kailangan mo ng mas kumplikadong expression upang ganap na mapatunayan ang papasok na data.
+Bilang kahalili, maaari mong gamitin ang `constraint` upang manu-manong magdagdag ng isang expression na dapat evaluate to true upang magpatuloy ang pagpapatupad. Ito ay kapaki-pakinabang kung ang pagbibigay ng pangalan ay hindi konsistent o kapag kailangan mo ng mas kumplikadong expression upang ganap na mapatunayan ang papasok na data.
```
Comment (kimerran) [inline content/account-data-matching.md]: `evaluate to` is still English

File: content/account-data-matching.md [hunk 5]
```diff
@@ -371,9 +371,9 @@ pub struct SecureWithdraw<'info> {
 }
 ```
 
-### 4. Subukan ang pagtuturo ng `secure_withdraw`
+### 4. Subukan ang instruction ng `secure_withdraw`
 
-Ngayon, subukan natin ang tagubiling `secure_withdraw` gamit ang dalawang pagsubok: isa na gumagamit ng `walletFake` bilang awtoridad at isa na gumagamit ng `wallet` bilang awtoridad. Inaasahan namin na ang unang invocation ay magbabalik ng error at ang pangalawa ay magtatagumpay.
+Ngayon, subukan natin ang tagubiling `secure_withdraw` gamit ang dalawang pagsubok: isa na gumagamit ng `walletFake` bilang awtoridad at isa na gumagamit ng `wallet` bilang awtoridad. Inaasahan namin na ang unang invocation ay mag return ng error at ang pangalawa ay magtatagumpay.
```
Comment (kimerran) [inline content/account-data-matching.md]: `tagubiling` to `instruction na` ?

File: content/account-data-matching.md [hunk 6]
```diff
@@ -454,4 +454,4 @@ Tulad ng iba pang mga aralin sa modyul na ito, ang iyong pagkakataon na magsanay
 
 Maglaan ng ilang oras upang suriin ang hindi bababa sa isang programa at tiyakin na ang mga wastong pagsusuri sa data ay nasa lugar upang maiwasan ang mga pagsasamantala sa seguridad.
 
-Tandaan, kung makakita ka ng bug o pagsasamantala sa programa ng ibang tao, mangyaring alertuhan sila! Kung makakita ka ng isa sa iyong sariling programa, siguraduhing i-patch ito kaagad.
\ No newline at end of file
+Tandaan, kung makakita ka ng bug o pagsasamantala sa programa ng ibang tao, mangyaring alertuhan sila! Kung makakita ka ng isa sa iyong sariling programa, siguraduhing i-patch ito kaagad
```
Comment (kimerran) [inline content/account-data-matching.md]: Should we remove the period on this?

File: content/anchor-cpi.md [hunk 7]
```diff
@@ -10,34 +10,34 @@ objectives:
...
+- Ang feature na **`cpi`** ng Anchor ay bumubuo ng mga function ng katulong ng CPI para sa paggamit ng mga tagubilin sa mga umiiral na Anchor program
 - Kung wala kang access sa mga function ng helper ng CPI, maaari mo pa ring gamitin ang `invoke` at `invoke_signed` nang direkta
-- Ginagamit ang **`error_code`** attribute macro para gumawa ng custom na Anchor Error
+- Ginagamit ang **`error_code`** attribute macro para gumawa ng custom na Anchor Error
 
-# Pangkalahatang-ideya
+# Overview
 
-Kung iisipin mo ang [unang CPI lesson](cpi.md), maaalala mo na ang paggawa ng mga CPI ay maaaring maging mahirap gamit ang vanilla Rust. Ang Anchor ay ginagawa itong medyo mas simple, lalo na kung ang program na iyong ini-invoke ay isa ring Anchor program na ang crate ay maa-access mo.
+Kung iisipin mo ang [first CPI lesson CPI lesson](cpi.md), maaalala mo na ang paggawa ng mga CPI ay maaaring maging mahirap gamit ang vanilla Rust. Ang Anchor ay ginagawa itong mas simple, lalo na kung ang program na iyong ini-invoke ay isa ring Anchor program na ang crate ay maa-access mo.
```
Comment (kimerran) [inline content/anchor-cpi.md]: duplicated 'CPI lesson'

File: content/anchor-cpi.md [hunk 8]
```diff
@@ -10,34 +10,34 @@ objectives:
...
 
-Sa araling ito, matututunan mo kung paano bumuo ng Anchor CPI. Matututuhan mo rin kung paano magtapon ng mga custom na error mula sa isang Anchor program para makapagsimula kang magsulat ng mas sopistikadong mga Anchor program.
+Sa araling ito, matututunan mo kung paano bumuo ng Anchor CPI. Matututuhan mo rin kung paano throw ng mga custom na error mula sa Anchor program para makapagsimula kang magsulat ng mas sopistikadong mga Anchor program.
```
Comment (kimerran) [inline content/anchor-cpi.md:21]: should be `i-throw` ?

File: content/anchor-cpi.md [hunk 9]
```diff
@@ -10,34 +10,34 @@ objectives:
...
 
-Ang uri ng `CpiContext` ay tumutukoy sa mga input na hindi argumento para sa mga cross program invocations:
+Ang uri ng `CpiContext` ay tumutukoy sa non-argument inputs para sa mga cross program invocations:
 
-- `accounts` - ang listahan ng mga account na kinakailangan para sa pagtuturo na ginagamit
+- `accounts` - listahan ng mga account na kinakailangan para sa instruction na ini-invoke
 - `remaining_accounts` - anumang natitirang account
 - `program` - ang program ID ng program na ini-invoke
-- `signer_seeds` - kung ang isang PDA ay pumipirma, isama ang mga buto na kinakailangan upang makuha ang PDA
+- `signer_seeds` - kung ang PDA ay nag signing, i-sasama nito ang mga seeds na kinakailangan upang ma-derived ang PD
```
Comment (kimerran) [inline content/anchor-cpi.md]: `PD` should be `PDA`

File: content/anchor-cpi.md [hunk 10]
```diff
@@ -71,7 +71,7 @@ pub fn new(
 }
 ```
 
-Gumagamit ka ng `CpiContext::new_with_signer` upang bumuo ng bagong instance kapag pumirma sa ngalan ng isang PDA para sa CPI.
+Ginagamit mo ang `CpiContext::new_with_signer` upang bumuo ng bagong instance kung kailan nag-signing on behalf of PDA para sa CPI.
```
Comment (kimerran) [inline content/anchor-cpi.md]: `mag-sign` ?

File: content/anchor-cpi.md [hunk 11]
```diff
@@ -119,11 +119,11 @@ Ipahayag lang ang dependency ng iyong program sa program na tinatawagan mo sa `C
...
+Ipinapakita ng module ng `cpi` ang mga instructions ng `callee` bilang isang Rust function na kumukuha bilang arguments ng `CpiContext` at anumang karagdagang instruction data. Gumagamit ang mga function na ito ng parehong format gaya ng mga function ng instruction sa iyong mga Anchor program, gamit lang ang `CpiContext` sa halip na `Context`. Ipinapakita din ng module na `cpi` ang mga istruktura ng account na kinakailangan para sa pagtawag sa mga tagubilin.
 
-Halimbawa, kung ang `callee` ay may tagubiling `do_something` na nangangailangan ng mga account na tinukoy sa `DoSomething` struct, maaari mong i-invoke ang `do_something` gaya ng sumusunod:
+Halimbawa, kung ang `callee` ay may instruction na `do_something` na nag-rerequires ng mga account na tinukoy sa `DoSomething` struct, maaari mong i-invoke ang `do_something` gaya ng sumusunod:
```
Comment (kimerran) [inline content/anchor-cpi.md]: `nag-rerequire` instead of `nag-rerequires`

File: content/anchor-cpi.md [hunk 12]
```diff
@@ -198,15 +198,15 @@ Kapag ang program na iyong tinatawagan ay *hindi* isang Anchor program, mayroong
...
     ```
 
-## Magtapon ng mga error sa Anchor
+## Throw errors in Anchor
 
-Malalim na kami sa Anchor sa puntong ito na mahalagang malaman kung paano gumawa ng mga custom na error.
+Malalim na tayo sa topic ng Anchor, sa puntong ito na mahalagang malaman kung paano gumawa ng mga custom errors.
 
-Sa huli, ibinabalik ng lahat ng program ang parehong uri ng error: [`ProgramError`](https://docs.rs/solana-program/latest/solana_program/program_error/enum.ProgramError.html). Gayunpaman, kapag nagsusulat ng isang programa gamit ang Anchor maaari mong gamitin ang `AnchorError` bilang abstraction sa ibabaw ng `ProgramError`. Ang abstraction na ito ay nagbibigay ng karagdagang impormasyon kapag nabigo ang isang programa, kabilang ang:
+Sa bandang huli, lahat ng program ay nag re-return nang parehong type ng error: [`ProgramError`](https://docs.rs/solana-program/latest/solana_program/program_error/enum.ProgramError.html). Gayunpaman, kapag nagsusulat ng isang programa gamit ang Anchor maaari mong gamitin ang `AnchorError` bilang abstraction sa ibabaw ng `ProgramError`. Ang abstraction na ito ay nagbibigay ng karagdagang impormasyon kapag nag-fails ang program, kabilang ang:
```
Comment (kimerran) [inline content/anchor-cpi.md]: `nag-fail`

File: content/anchor-cpi.md [hunk 13]
```diff
@@ -255,7 +255,7 @@ pub enum MyError {
 }
 ```
 
-Bilang kahalili, maaari mong gamitin ang [require](https://docs.rs/anchor-lang/latest/anchor_lang/macro.require.html) macro para pasimplehin ang mga bumabalik na error. Ang code sa itaas ay maaaring i-refactor sa sumusunod:
+Bukod doon, maaari rin gamitin ang [require](https://docs.rs/anchor-lang/latest/anchor_lang/macro.require.html) macro para pasimplehin ang returning ng error. Ang code sa itaas ay maaaring i-refactored sa sumusunod:
```
Comment (kimerran) [inline content/anchor-cpi.md]: `i-refactor`

File: content/anchor-cpi.md [hunk 14]
```diff
@@ -303,7 +303,7 @@ anchor-spl = "0.25.0"
 
 ### 3. Magsimula ng reward token
 
-Susunod, mag-navigate sa `lib.rs` at gumawa ng tagubilin para makapagsimula ng bagong token mint. Ito ang magiging token na mined sa tuwing ang isang user ay umalis ng isang review. Tandaan na hindi namin kailangang isama ang anumang custom na lohika ng pagtuturo dahil ang pagsisimula ay maaaring ganap na mahawakan sa pamamagitan ng mga hadlang sa Anchor.
+Kasunod nito, mag-navigate sa `lib.rs` at gumawa ng instruction para makapag-initialize ng bagong token mint. Ito ang magiging token na mined sa tuwing ang isang user ay nag-iwann ng review. Tandaan na hindi namin kailangang isama ang anumang custom na lohika ng pagtuturo dahil ang pagsisimula ay maaaring ganap na mahawakan sa pamamagitan ng mga hadlang sa Anchor.
```
Comment (kimerran) [inline content/anchor-cpi.md]: `nag-iwan`

File: content/anchor-cpi.md [hunk 15]
```diff
@@ -336,11 +336,11 @@ pub struct InitializeMint<'info> {
 }
 ```
 
-Maaaring may ilang mga hadlang sa itaas na hindi mo pa nakikita. Ang pagdaragdag ng `mint::decimals` at `mint::authority` kasama ng `init` ay nagsisiguro na ang account ay masisimulan bilang isang bagong token mint na may naaangkop na mga decimal at mint authority set.
+Maaaring may ilang mga constraints sa itaas na hindi mo pa nakikita. Ang pagdaragdag ng `mint::decimals` at `mint::authority` kasama ng `init` ay nagsisiguro na ang account ay ma-initialized bilang isang bagong token mint na may naaangkop na mga decimal at mint authority set.
```
Comment (kimerran) [inline content/anchor-cpi.md]: `ma-initialize`

File: content/anchor-cpi.md [hunk 16]
```diff
@@ -397,14 +397,14 @@ pub struct AddMovieReview<'info> {
 }
 ```
 
-Muli, ang ilan sa mga hadlang sa itaas ay maaaring hindi pamilyar sa iyo. Ang `associated_token::mint` at `associated_token::authority` na mga hadlang kasama ang `init_if_needed` constraint ay nagsisiguro na kung ang account ay hindi pa nasisimulan, ito ay pasisimulan bilang isang nauugnay na token account para sa tinukoy na mint at awtoridad.
+Muli, ang ilan sa mga constraints sa itaas ay maaaring hindi pamilyar sa iyo. Ang `associated_token::mint` at `associated_token::authority` na mga constraint kasama ang `init_if_needed` constraint ay nagsisiguro na kung ang account ay hindi pa na-initialized, ito ay pasisimulan bilang isang nauugnay na token account para sa tinukoy na mint at authority.
```
Comment (kimerran) [inline content/anchor-cpi.md]: `na-initialize`

File: content/anchor-cpi.md [hunk 17]
```diff
@@ -397,14 +397,14 @@ pub struct AddMovieReview<'info> {
...
 - Gumawa ng CPI sa pagtuturo ng `mint_to` ng token program gamit ang mint authority PDA bilang isang pumirma. Tandaan na gagawa kami ng 10 token sa user ngunit kailangan naming mag-adjust para sa mint decimal sa pamamagitan ng paggawa nitong `10*10^6`.
 
-Sa kabutihang palad, maaari naming gamitin ang `anchor_spl` crate upang ma-access ang mga function at uri ng helper tulad ng `mint_to` at `MintTo` para sa pagbuo ng aming CPI sa Token Program. Ang `mint_to` ay kumukuha ng `CpiContext` at integer bilang mga argumento, kung saan kinakatawan ng integer ang bilang ng mga token na gagawin. Maaaring gamitin ang `MintTo` para sa listahan ng mga account na kailangan ng pagtuturo ng mint.
+Sa kabutihang palad, maaari naming gamitin ang `anchor_spl` crate upang ma-access ang mga function at uri ng helper tulad ng `mint_to` at `MintTo` para sa pagbuo ng aming CPI sa Token Program. Ang `mint_to` ay kumukuha ng `CpiContext` at integer bilang isang arguments, kung saan kinakatawan ng integer ang bilang ng mga token na gagawin. Maaaring gamitin ang `MintTo` para sa listahan ng mga account na kailangan ng instruction ng mint.
```
Comment (kimerran) [inline content/anchor-cpi.md]: `isang argument`

File: content/bump-seed-canonicalization.md [hunk 18]
```diff
@@ -103,9 +103,9 @@ pub fn set_value_secure(
...
+Nagbibigay ang Anchor ng maginhawang paraan upang mag-derive ng mga PDAs sa struct ng account validation gamit ang mga constraints na `seeds` at `bump`. Ang mga ito ay maaaring isama pa sa `init` na constraint upang ma-initialize ang account sa intended na address. Upang maprotektahan ang programa mula sa kahinaan na tinatalakay namin sa buong araling ito, hindi ka pinapayagan ng Anchor na        mag-initialize ng isang account sa isang PDA gamit ang anumang bagay maliban sa canonical bump. Sa halip, gumagamit ito ng `find_program_address` para makuha ang PDA at pagkatapos ay isagawa ang initialization.
```
Comment (kimerran) [inline content/bump-seed-canonicalization.md]: un-necessary space before `mag-initialize`

File: content/bump-seed-canonicalization.md [hunk 19]
```diff
@@ -211,9 +211,9 @@ pub struct Data {
...
+Kung hindi mo tutukuyin ang bump sa `bump` constraint, gagamitin pa rin ni Anchor ang `find_program_address` para makuha ang PDA gamit ang canonical bump. Bilang resulta, ang iyong instruction ay magkakaroon ng variable na halaga ng compute budget. Dapat itong gamitin ng mga program na nasa panganib na lumampas sa kanilang compute budget dahil may posibilidad na ang badyet ng programa ay maaaring paminsan-minsan at hindi mahuhulaan na lumampas.
 
-Sa kabilang banda, kung kailangan mo lang i-verify ang address ng isang PDA na ipinasa nang hindi sinisimulan ang isang account, mapipilitan kang hayaan ang Anchor na makuha ang canonical bump o ilantad ang iyong programa sa mga hindi kinakailangang panganib. Kung ganoon, mangyaring gamitin ang canonical bump sa kabila ng bahagyang marka laban sa pagganap.
+Sa kabilang banda, kung kailangan mo lang i-verify ang address ng isang PDA na ipinasa nang hindi nag-initializing ng account, mapipilitan kang hayaan ang Anchor na ma-derive ang canonical bump o ilantad ang iyong programa sa mga hindi kinakailangang panganib. Kung ganoon, mangyaring gamitin ang canonical bump sa kabila ng bahagyang marka laban sa pagganap.
```
Comment (kimerran) [inline content/bump-seed-canonicalization.md]: `nag-initialize`

File: content/cpi.md [hunk 20]
```diff
@@ -128,19 +128,19 @@ Malamang na bago sa iyo ang [`extend_from_slice`](https://doc.rust-lang.org/allo
...
 
-Maaari mong kopyahin ang bawat object na `account_info` na kailangan mong ipasa sa CPI gamit ang [`Clone`](https://docs.rs/solana-program/1.10.19/solana_program/account_info/struct.AccountInfo.html# impl-Clone) na katangian na ipinatupad sa `account_info` struct sa `solana_program` crate. Ang katangiang ito ng `Clone` ay nagbabalik ng kopya ng [`account_info`](https://docs.rs/solana-program/1.10.19/solana_program/account_info/struct.AccountInfo.html) instance.
+Maaari mong kopyahin ang bawat object na `account_info` na kailangan mong ipasa sa CPI gamit ang [`Clone`](https://docs.rs/solana-program/1.10.19/solana_program/account_info/struct.AccountInfo.html# impl-Clone) na katangian na ipinatupad sa `account_info` struct sa `solana_program` crate. Ang trait ng `Clone` ay nag re-returns ng kopya ng [`account_info`](https://docs.rs/solana-program/1.10.19/solana_program/account_info/struct.AccountInfo.html) instance.
```
Comment (kimerran) [inline content/cpi.md]: `nag-rereturn`

File: content/cpi.md [hunk 21]
```diff
@@ -355,9 +355,9 @@ invoke_signed(
...
+Tandaan na gumagamit kami ng `invoke_signed` at hindi `invoke` dito. Ang Token program ay nangangailangan ng `mint_auth` account upang mag-sign para sa transaksyong ito. Dahil ang `mint_auth` account ay isang PDA, tanging ang program kung saan ito hinango ay maaaring mag-sign sa ngalan nito. Kapag tinawag ang `invoke_signed`, ang runtime ng Solana ay tatawag ng `create_program_address` kasama ang mga seeds at bump na ibinigay at pagkatapos ay ihahambing ang nagmula na address sa lahat ng mga address ng ibinigay na `AccountInfo` object. Kung ang alinman sa mga address ay tumutugma sa nagmula na address, alam ng runtime na ang katugmang account ay isang PDA ng program na ito at na ang program ay signing sa transaksyong ito para sa account na ito.
```
Comment (kimerran) [inline content/cpi.md]: `mag-sign`

File: content/deserialize-custom-data.md [hunk 22]
```diff
@@ -64,7 +64,7 @@ const [pda, bump] = await web3.PublicKey.findProgramAddress(
 )
 ```
 
-Gayundin, kapag mayroong maramihang mga account sa bawat user, maaaring gumamit ang isang program ng isa o higit pang karagdagang mga binhi upang lumikha at tumukoy ng mga account. Halimbawa, sa isang note-taking app ay maaaring mayroong isang account sa bawat tala kung saan ang bawat PDA ay hinango kasama ang pampublikong key ng user at ang pamagat ng tala.
+Gayundin, kapag mayroong maramihang mga account sa bawat user, maaaring gumamit ang isang program ng isa o higit pang karagdagang mga seeds upang lumikha at tumukoy ng mga account. Halimbawa, sa isang note-taking app ay maaaring mayroong isang account sa bawat tala kung saan ang bawat PDA ay hinango kasama ang public key ng user at ang title ng note’s.
```
Comment (kimerran) [inline content/deserialize-custom-data.md]: `note`

File: content/deserialize-custom-data.md [hunk 23]
```diff
@@ -76,9 +76,9 @@ const [pda, bump] = await web3.PublicKey.findProgramAddress(
 )
 ```
 
-### Pagkuha ng Maramihang Programa Accounts
+### Getting Multiple Program Accounts
 
-Bilang karagdagan sa pagkuha ng mga address, maaari mong kunin ang lahat ng account na ginawa ng isang program gamit ang `connection.getProgramAccounts(programId)`. Nagbabalik ito ng hanay ng mga bagay kung saan ang bawat bagay ay may `pubkey` na property na kumakatawan sa pampublikong key ng account at isang `account` na property na may uri ng `AccountInfo`. Maaari mong gamitin ang property na `account` para makuha ang data ng account.
+Bilang karagdagan sa pagkuha ng mga address, maaari mong kunin ang lahat ng account na ginawa ng isang program gamit ang `connection.getProgramAccounts(programId)`. Nag re-returns ito ng array of objects kung saan ang bawat bagay ay may `pubkey` na property na kumakatawan sa public key ng account at isang `account` na property na may type ng `AccountInfo`. Maaari mong gamitin ang property na `account` para makuha ang data ng account.
```
Comment (kimerran) [inline content/deserialize-custom-data.md]: `nag re-return`

File: content/deserialize-instruction-data.md [hunk 24]
```diff
@@ -12,18 +12,18 @@ objectives:
 
 # TL;DR
 
-- Karamihan sa mga programa ay sumusuporta sa **maraming discrete na mga tagubilin** - magpapasya ka kapag isinusulat mo ang iyong programa kung ano ang mga tagubiling ito at kung anong data ang dapat kasama sa kanila
-- Ang mga kalawang **enums** ay kadalasang ginagamit upang kumatawan sa mga hiwalay na tagubilin ng programa
+- Karamihan sa mga programa ay sumusuporta sa **multiple discrete instructions** - magpapasya ka kapag isinusulat mo ang iyong programa kung ano ang mga instructions ito at kung anong data ang dapat kasama sa kanila
+- Ang mga kalawang **enums** ay kadalasang ginagamit upang kumatawan sa mga hiwalay na instructions ng programa
```
Comment (kimerran) [inline content/deserialize-instruction-data.md]: kalawang to `Rust` please also check if we still have several instance of `kalawang`

File: content/env-variables.md [hunk 25]
```diff
@@ -151,7 +151,7 @@ pub mod my_program {
 
 Sa halimbawang ito, ginagamit ng `test_function` ang `cfg!` na macro upang suriin ang halaga ng feature na `local-testing` sa runtime. Kung ang feature na `local-testing` ay pinagana, ang unang code path ay isasagawa. Kung ang feature na `local-testing` ay hindi pinagana, ang pangalawang code path ang ipapatupad sa halip.
 
-## Mga tagubilin para sa admin lamang
+### Admin-only instructions
```
Comment (kimerran) [inline content/env-variables.md]: remove the extra `#`

File: content/env-variables.md [hunk 26]
```diff
@@ -694,14 +694,14 @@ describe("config", () => {
...
+Susunod, i-update ang test file na may tatlo pang tests na testing that:
 
 1. Ang program config account ay nasimulan nang tama
-2. Ang tagubilin sa pagbabayad ay gumagana ayon sa nilalayon
+2. Ang instruction sa pagbabayad ay gumagana ayon sa nilalayon
 3. Ang config account ay maaaring matagumpay na ma-update ng admin
 4. Ang config account ay hindi maaaring ma-update ng iba maliban sa admin
 
-Sinisimulan ng unang pagsubok ang program config account at bini-verify na ang tamang bayad ay nakatakda at na ang tamang admin ay naka-store sa program config account.
+I-initializes ng unang test ang program config account at bini-verify na ang tamang bayad ay nakatakda at na ang tamang admin ay naka-store sa program config account.
```
Comment (kimerran) [inline content/env-variables.md]: I think `Ini-initialize` is better

File: content/intro-to-anchor-frontend.md [hunk 27]
```diff
@@ -503,14 +501,14 @@ Kung kailangan mo ng mas maraming oras sa proyektong ito para maging komportable
...
 
-1. Bumuo at mag-deploy ng bagong program na nagpapatupad ng pagtuturo ng `decrement`
+1. Bumuo at mag-deploy ng bagong program na nagpapatupad ng instruction ng `decrement`
 2. I-update ang IDL file sa frontend gamit ang isa mula sa iyong bagong program
 3. I-update ang `programId` gamit ang isa mula sa iyong bagong program
 
 Kung kailangan mo ng tulong, huwag mag-atubiling sumangguni sa program na ito [dito](https://github.com/Unboxed-Software/anchor-counter-program/tree/solution-decrement).
 
-Subukang gawin ito nang nakapag-iisa kung kaya mo! Ngunit kung natigil ka, huwag mag-atubiling sumangguni sa [solution code](https://github.com/Unboxed-Software/anchor-ping-frontend/tree/solution-decrement).
+Subukang gawin ito nang mag-iisa kung kaya mo! Ngunit kung nahihirapan ka, huwag mag-atubiling sumangguni sa [solution code](https://github.com/Unboxed-Software/anchor-ping-frontend/tree/solution-decrement).
```
Comment (kimerran) [inline content/intro-to-anchor-frontend.md]: `mag-isa`

File: content/intro-to-anchor.md [hunk 28]
```diff
@@ -183,21 +183,21 @@ Signer.info.is_signer == true
 
 ### `Program`
 
-Ang uri ng `Program` ay nagpapatunay na ang account ay isang partikular na programa.
+Ang type ng `Program` ay nag-validates na ang account ay isang partikular na programa.
```
Comment (kimerran) [inline content/intro-to-anchor.md]: `nag-validate`

File: content/intro-to-reading-data.md [hunk 29]
```diff
@@ -130,13 +130,13 @@ async function getBalanceUsingWeb3(address: PublicKey): Promise<number> {
 }
 ```
 
-Ang ibinalik na balanse ay nasa fractional SOL na tinatawag na lamports. Ang isang lamport ay kumakatawan sa 0.000000001 SOL. Karamihan sa mga oras kapag nakikitungo sa SOL ang system ay gagamit ng mga lamport sa halip na SOL. Nagbibigay ang Web3.js ng patuloy na `LAMPORTS_PER_SOL` para sa paggawa ng mabilis na mga conversion.
+Ang na-returned na balanse ay nasa fractional SOL na tinatawag na lamports. Ang isang lamport ay kumakatawan sa 0.000000001 SOL. Karamihan sa mga oras kapag nakikitungo sa SOL ang system ay gagamit ng mga lamport sa halip na SOL. Nagbibigay ang Web3.js ng patuloy na `LAMPORTS_PER_SOL` para sa paggawa ng mabilis na mga conversion.
```
Comment (kimerran) [inline content/intro-to-reading-data.md]: `na-return`

File: content/intro-to-writing-data.md [hunk 30]
```diff
@@ -134,20 +134,20 @@ export type TransactionInstructionCtorFields = {
 };
 ```
 
-Alinsunod sa kahulugan sa itaas, ang bagay na ipinasa sa `TransactionInstruction` constructor ay nangangailangan ng:
+Alinsunod sa kahulugan sa itaas, ang bagay na ipinasa sa `TransactionInstruction` constructor ay requires ng:
```
Comment (kimerran) [inline content/intro-to-writing-data.md]: `nag-require`

File: content/intro-to-writing-data.md [hunk 31]
```diff
@@ -134,20 +134,20 @@ export type TransactionInstructionCtorFields = {
...
+Ang field ng `programId` ay medyo nagpapaliwanag sa sarili: ito ang public key na nauugnay sa programa. Kakailanganin mong malaman ito nang maaga sa pagtawag sa programa sa parehong paraan na kailangan mong malaman ang public key ng isang taong gusto mong padalhan ng SOL.
 
-Ang hanay ng `keys` ay nangangailangan ng kaunting paliwanag. Ang bawat bagay sa array na ito ay kumakatawan sa isang account na babasahin o isusulat sa panahon ng pagpapatupad ng isang transaksyon. Nangangahulugan ito na kailangan mong malaman ang pag-uugali ng program na iyong tinatawagan at tiyaking ibibigay mo ang lahat ng kinakailangang mga account sa array.
+Ang array ng `keys` ay requires ng kaunting paliwanag. Ang bawat bagay sa array na ito ay kumakatawan sa isang account na babasahin o isusulat sa panahon ng pagpapatupad ng isang transaksyon. Nangangahulugan ito na kailangan mong malaman ang pag-uugali ng program na iyong tinatawagan at tiyaking ibibigay mo ang lahat ng kinakailangang mga account sa array.
```
Comment (kimerran) [inline content/intro-to-writing-data.md]: `nag-require`

File: content/account-data-matching.md (hunk 1 above)
Comment (JessieJaradal) [reply content/account-data-matching.md]: Ahyt sorry sir, I'll correct the typo.

File: content/account-data-matching.md (hunk 4 above)
Comment (JessieJaradal) [reply content/account-data-matching.md]: Hello sir, Instead of evaluate to, I'll use "mag-evaluate to" para mag makes sense sya sa tag-lish. Hindi po kasi tugma kapag "suriin" yung gamit na word.

File: content/account-data-matching.md (hunk 5 above)
Comment (JessieJaradal) [reply content/account-data-matching.md]: I'll correct this typo mistake, sir.

File: content/account-data-matching.md (hunk 6 above)
Comment (JessieJaradal) [reply content/account-data-matching.md]: I'll correct this typo mistake, sir.

File: content/deserialize-instruction-data.md (hunk 24 above)
Comment (JessieJaradal) [reply content/deserialize-instruction-data.md]: Noted po, sir.

File: content/env-variables.md (hunk 26 above)
Comment (JessieJaradal) [reply content/env-variables.md]: Noted po, sir.

File: content/intro-to-writing-data.md (hunk 31 above)
Comment (JessieJaradal) [reply content/intro-to-writing-data.md]: Noted po, sir.
